docs/network_new/network_module.md

```diff
@@ -0,0 +1,76 @@
+# Network module
+
+## ZBus
+
+Network module is available on zbus over the following channel
+
+| module | object | version |
+|--------|--------|---------|
+| network|[network](#interface)| 0.0.1|
+
+## Home Directory
+
+network keeps some data in the following locations
+| directory | path|
+|----|---|
+| root| `/var/cache/modules/network`|
+
+## Interface
+
+```go
+//Networker is the interface for the network module
+type Networker interface {
+  ApplyNetResource(Network) (string, error)
+  DeleteNetResource(Network) error
+  Namespace(NetID) (string, error)
+}
+```
+
+## Zero-OS networking
+
+### Some First Explanations
```

Comment:
This sounds like redundant with the rest of the documentation. Let's get rid of this.